wordpress-mobile/WordPress-Android
# Add more types to media picker

Fixes #12636 

This PR adds an option to pick Audio files and documents to the Media picker. The audio files are selected from the content provider. The documents are from the Downloads folder. The media picker should only show supported types. I've replaced the "Choose from device" option in the "Media Browser activity" with the media picker as part of this PR.

I've removed ID as the main item identifier in this PR and replaced it with URI. The reason is that the documents are not loaded through content provider and don't have IDs. 

To test:
- Enable the flag
- Go to My Site/Media
- Click on +/Choose from device
- Notice the media picker is shown
- Notice you see Audio files and documents in the list, ordered by the change date
- Notice that the selector sometimes overlaps the file name -> that will be fixed later
- Select a few files
- Confirm selection
- Files get uploaded

PR submission checklist:
